### PR TITLE
Language extensions

### PR DESCRIPTION
**Pull request by AvinashSeelam9**
## What does this PR do and why?
This PR provides Cloud9 Support for Language Extensions Lab
_Describe in detail what your merge request does and why._
This PR provides Cloud9 Support for Language Extensions Lab
## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/intermediate/templates/language-extensions/index.md [hunk 1]
```diff
@@ -103,9 +140,9 @@ Now that you have your EC2 instance running, you choose to monitor it by creatin
 
 A dashboard body is a string in JSON format: for more information, see [Dashboard Body Structure and Syntax](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html). When you describe a CloudWatch dashboard with CloudFormation, you specify a JSON string that contains keys and values, such as:
 
-```yaml
+:::code{language=yaml showLineNumbers=true showCopyAction=true}
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:143]: should this be JSON?

File: content/intermediate/templates/language-extensions/index.md [hunk 2]
```diff
@@ -140,25 +177,47 @@ You'll now update the `language-extensions.yaml` template to add a CloudWatch da
...
 4. Under **Prerequisite - Prepare template,** select **Replace current template** and choose **Upload a template file**. Select **Choose file**, and supply the `language-extensions.yaml` template you updated earlier, and then choose **Next**.
 5. On **Specify Stack details** page, leave the configuration as it is. Choose **Next**.
 6. On **Configure Stack options**, leave the configuration as it is. Choose **Next**.
 7. On **Review** page, review the contents of the page. At the bottom of the page, choose to acknowledge all the capabilities shown in the **Capabilities and transforms** section.
 8. Choose **Submit**. Refresh the stack creation page until you see the stack in the `UPDATE_COMPLETE` status.
-9. Navigate to the [CloudWatch console](https://console.aws.amazon.com/cloudwatch/). From the right navigation panel, choose **Dashboards**. You should see the dashboard you just created.
-
+::::
+:::::
+Navigate to the [CloudWatch console](https://console.aws.amazon.com/cloudwatch/). From the right navigation panel, choose **Dashboards**. You should see the dashboard you just created.
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:220]: Should we add in some additional guidance around inspecting the dashboard source:
* open dashboard
* choose Actions
* choose View/edit source
You should see the JSON for the dashboard that matches your YAML from `language-extensions.yaml`

File: content/intermediate/templates/language-extensions/index.md [hunk 3]
```diff
@@ -181,19 +240,68 @@ Refer to the [CloudWatch Dashboard structure](https://docs.aws.amazon.com/Amazon
...
+1. Repeat steps (1-2) above to delete the stack `cfn-workshop-language-extensions-solution`.
+::::
+::::tab{id="LocalDevelopment" label="Local development"}
 1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
-2. On the **Stacks** page in the CloudFormation console, select the stack you created in **Part 1:** `language-extensions`.
-3. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
-4. On the **Stacks** page in the CloudFormation console, select the stack you created in **Challenge** section: `language-extensions-solution`.
-5. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
-
+1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Part 1:** `language-extensions`.
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:299]:
```suggestion
1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Part 1:** `cfn-workshop-language-extensions`.
```

File: content/intermediate/templates/language-extensions/index.md [hunk 4]
```diff
@@ -181,19 +240,68 @@ Refer to the [CloudWatch Dashboard structure](https://docs.aws.amazon.com/Amazon
...
+1. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
+1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Challenge** section: `language-extensions-solution`.
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:301]:
```suggestion
1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Challenge** section: `cfn-workshop-language-extensions-solution`.
```

File: content/intermediate/templates/language-extensions/index.md [hunk 5]
```diff
@@ -181,19 +240,68 @@ Refer to the [CloudWatch Dashboard structure](https://docs.aws.amazon.com/Amazon
...
+::::
+::::tab{id="LocalDevelopment" label="Local development"}
 1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
-2. On the **Stacks** page in the CloudFormation console, select the stack you created in **Part 1:** `language-extensions`.
-3. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
-4. On the **Stacks** page in the CloudFormation console, select the stack you created in **Challenge** section: `language-extensions-solution`.
-5. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
-
+1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Part 1:** `language-extensions`.
+1. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:300]:
```suggestion
1. In the stack details pane, choose **Delete**. Select **Delete** when prompted.
```

File: content/intermediate/templates/language-extensions/index.md [hunk 6]
```diff
@@ -181,19 +240,68 @@ Refer to the [CloudWatch Dashboard structure](https://docs.aws.amazon.com/Amazon
...
+1. On the **Stacks** page in the CloudFormation console, select the stack you created in **Challenge** section: `language-extensions-solution`.
+1. In the stack details pane, choose **Delete**. Select **Delete stack** when prompted.
```
Comment (cjhar) [inline content/intermediate/templates/language-extensions/index.md:302]:
```suggestion
1. In the stack details pane, choose **Delete**. Select **Delete** when prompted.
```

**Review by cjhar — COMMENTED** [review]
Thanks, just some small corrections to the clean up section. Also we could ask the user to inspect the dashboard source to verify the template created properly

File: content/intermediate/templates/language-extensions/index.md (hunk 1 above)
Comment (AvinashSeelam9) [reply content/intermediate/templates/language-extensions/index.md]: yes. I have updated it

**Review by cjhar — COMMENTED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
Nice one! LGMT

**Pull request merged**